.travis.yml

```diff
@@ -1,13 +1,11 @@
 language: php
 
 php:
-  - 7.1
-  - 7.2
-  - 7.3
+  - '7.1'
```

Comment:
Reason for this change?

Reply:
Spec tells me the items there should be strings, and they're technically numbers now.

https://docs.travis-ci.com/user/languages/php/#choosing-php-versions-to-test-against